src/DatePicker/DatePicker.js

```diff
@@ -171,17 +167,20 @@ class DatePicker extends React.Component {
     style: {},
   };
 
+
   static contextTypes = {
     muiTheme: React.PropTypes.object.isRequired,
   };
 
   state = {
-    date: undefined,
+    date: null,
+    dialogDate: null,
```

Comment:
This was there since before. I haven't touched this piece of code. Let me take a look into it though.